test/include/seqan3/test/testing_list.hpp

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2018, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2018, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <seqan3/core/metafunction/template_inspection.hpp>
+
+namespace seqan3
```

Comment:
IMPOV this should be detail and no public interface? 

And maybe call this file utility or basic since it is just some convenience helper of which we might implement more and not each of them should go into a separate file.

Reply:
My thought was, that it is public in the test environment, because it's only accessible in there.

Reply:
True, if you see it like that. But a whole file for this tiny helper is still megalomaniac :grin:  (yes I googled this word)